Add basic docs infrastructure (#5)
plone/plone.restapi-client: docs/index.md

---
myst:
  html_meta:

>     "description": "Documentation for the Plone RESTAPI JavaScript client - JS framework agnostic library based on TanStack Query"

Reviewer: ```suggestion
    "description": "Documentation for the Plone REST API JavaScript client - a framework agnostic library based on TanStack Query"
```

>     "property=og:description": "Documentation for the Plone RESTAPI JavaScript client - JS framework agnostic library based on TanStack Query"

Reviewer: ```suggestion
    "property=og:description": "Documentation for the Plone RESTAPI JavaScript client - a framework agnostic library based on TanStack Query"
```

>     "property=og:title": "Plone RESTAPI JavaScript Client"

Reviewer: ```suggestion
    "property=og:title": "Plone REST API JavaScript Client"
```

>     "keywords": "Plone, RESTAPI, client, JavaScript"

Reviewer: ```suggestion
    "keywords": "Plone, REST, API, client, JavaScript"
```

---

> # Plone RESTAPI JavaScript Client

Reviewer: ```suggestion
# Plone REST API JavaScript Client
```

> This is the documentation for the Plone RESTAPI JavaScript Client

Reviewer: ```suggestion
This is the documentation for the Plone REST API JavaScript Client
```
